docs/installation.md

```diff
@@ -34,42 +34,28 @@ $ python3 amundsen_application/wsgi.py
-  * In your local versions of each library, update the `LOCAL_HOST` in the `LocalConfig` with the IP used for the `default` docker machine. You can see the IP in the `URL` outputted from running `docker-machine ls`.
-7. Start all of the services using:
-```bash
-# in ~/<your-path-to-cloned-repo>/amundsenfrontendlibrary
-$ docker-compose -f docker-amundsen.yml up
-```
-8. Ingest dummy data into Neo4j by doing the following:
-  * Clone [amundsendatabuilder](https://github.com/lyft/amundsendatabuilder).
-  * Update the `NEO4J_ENDPOINT` and `Elasticsearch host` in [sample_data_loader.py](https://github.com/lyft/amundsendatabuilder/blob/master/example/scripts/sample_data_loader.py) and replace `localhost` with the IP used for the `default` docker machine. You can see the IP in the `URL` outputted from running `docker-machine ls`.
-  * Run the following commands:
+1. Install `docker` and  `docker-compose`.
+2. Setup your local environment.
+   * Clone [this repo](https://github.com/lyft/amundsenfrontendlibrary) or download the [docker-amundsen.yml](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docker-amundsen.yml) file directly.
+3. Enter the directory where the `docker-amundsen.yml` file is and then:
     ```bash
-    # in ~/<your-path-to-cloned-repo>/amundsendatabuilder
-    $ virtualenv -p python3 venv3
-    $ source venv3/bin/activate  
+    $ docker-compose -f docker-amundsen.yml up -d
+    ```
+4. Ingest dummy data into Neo4j by doing the following:
+   * Clone [amundsendatabuilder](https://github.com/lyft/amundsendatabuilder).
+   * Run the following commands in the `amundsenddatabuilder` directory:
+   ```bash
+    $ python3 -m venv venv
+    $ source venv/bin/activate  
     $ pip3 install -r requirements.txt
-    $ python setup.py install      
-    $ python example/scripts/sample_data_loader.py
+    $ python3 setup.py install
+    $ python3 example/scripts/sample_data_loader.py
+   ```
+5. You can verify dummy data has been ingested by viewing in Neo4j by visiting [`http://localhost:7474/browser/`](http://localhost:7474/browser/) and run `MATCH (n:Table) RETURN n LIMIT 25` in the query box. You should see two tables
+   1. `hive.test_schema.test_table1`
+   2. `dynamo.test_schema.test_table2`
+6. View UI at [`http://localhost:5000`](http://localhost:5000) and try to search `test`, it should return some result.
+7. From where the `docker-amundsen.yml` file is, run the following command when done:
+    ```bash
+    $ docker-compose -f docker-amundsen.yml down
     ```
-9. Verify dummy data has been ingested by viewing in Neo4j by visiting `http://YOUR-DOCKER-HOST-IP:7474/browser/` and run `MATCH (n:Table) RETURN n LIMIT 25` in the query box. You should see two tables -- `hive.test_schema.test_table1` and `dynamo.test_schema.test_table2`.
-10. View UI at `http://YOUR-DOCKER-HOST-IP:5000/table_detail/gold/hive/test_schema/test_table1` or `/table_detail/gold/dynamo/test_schema/test_table2`
```

Comment:
Verifying the UI at these urls has value because it checks that the  metadata service is running as expected. To summarize -- the old Step 9 verified Neo4j had the dummy data, the old Step 10 verified that the metadata service is correctly returning that data, and the old Step 11 verified that the search service can correctly search for that data.

I would opt for still keeping the step to view these urls to verify the metadata service, however we can just simply modify your new Step 6 to read:

> 6. View UI at [`http://localhost:5000`](http://localhost:5000) and try to search `test`, it should return some results. Click on one of the results, it should navigate to a page that displays the metadata for that resource.

Reply:
Step 5 still has them do that

Reply:
Step 5 asks them to verify that the data exists in Neo4 at `http://localhost:7474/browser/`. We would still want folks to verify that the application UI itself correctly displays that metadata from Neo4j at `http://localhost:5000/table_detail/...`

Reply:
I'm actually more tempted to remove step 5 altogether as well since it's not really part of a quick start. Those steps seem more appropriate for debugging.

Reply:
We intended for this documentation to include the steps to set up the tool and verify that it works. I would consider the last 3 steps of the old doc to be verification rather than debugging. 

If separating the setup from the verification is desirable then we can split these steps into two separate sections as opposed to deleting the instructions that help verify things work as expected. However I do prefer to keep them in the same flow, so that our installation instructions follow the pattern: setup --> verify it works --> teardown.